tools/apksize/apksize.gradle

```diff
@@ -50,6 +51,35 @@ task uploadMeasurements(type: UploadMeasurementsTask) {
     uploader = "https://storage.googleapis.com/firebase-engprod-metrics/upload_tool.jar"
 }
 
+task generateCoverageMeasurements(type: GenerateCoveragePercentsTask, group: 'Coverage') {
+
+    // TODO(yifany@): Way too cumbersome.
+    dependsOn ':firebase-common:checkCoverage'
+    dependsOn ':firebase-database:checkCoverage'
+    dependsOn ':firebase-database-collection:checkCoverage'
+    dependsOn ':firebase-firestore:checkCoverage'
+    dependsOn ':firebase-functions:checkCoverage'
+    dependsOn ':firebase-inappmessaging-display:checkCoverage'
+    dependsOn ':firebase-storage:checkCoverage'
+
+    packages = ["firebase-common",
+                "firebase-database",
+                "firebase-database-collection",
+                "firebase-firestore",
+                "firebase-functions",
+                "firebase-inappmessaging-display",
+                "firebase-storage"]
+
+    reportFile = file("$buildDir/coverage-report.json")
+}
+
+task uploadCoverageMeasurements(type: UploadMeasurementsTask, group: 'Coverage') {
```

Comment:
Maybe add a TODO to move this out of apksize gradle? How hard would it be to move it out now?

Reply:
Done.

Reply:
The plugins in buildSrc are independent of tools/apksize. I think it's sensible to create tools/measurement and move apksize under that. This can be done in a future pull request, but I don't think it would be that difficult to move this new task to measurement/.

In fact, there's no code for this like there is with APK size, so it can probably live in top level measurement.